lib/matplotlib/animation.py

```diff
@@ -246,7 +246,7 @@ def frame_size(self):
         width_inches, height_inches = self.fig.get_size_inches()
         return width_inches * self.dpi, height_inches * self.dpi
 
-    def setup(self, fig, outfile, dpi, *args):
+    def setup(self, fig, outfile, dpi, *args, **kwargs):
```

Comment:
can you remove both args and kwargs from this signature?  We do not use either of them so having them will just mask errors.